scripts/general/seedUmaToAccounts.js

```diff
@@ -0,0 +1,125 @@
+const hre = require("hardhat");
```

Comment:
this file does more now than just seed uma, and its a bit more specific to oo, i wonder if this should be left alone and copied into oo specific scripts